templates/default/redis.conf.erb

```diff
@@ -250,6 +250,69 @@ dir <%=@datadir%>
 #    but to INFO and SLAVEOF.
 #
 slave-serve-stale-data <%=@slaveservestaledata%>
+<% if @version[:major].to_i == 2 && @version[:minor].to_i >= 6 %>
+
+# You can configure a slave instance to accept writes or not. Writing against
+# a slave instance may be useful to store some ephemeral data (because data
+# written on a slave will be easily deleted after resync with the master) but
+# may also cause problems if clients are writing to it because of a
+# misconfiguration.
+#
+# Since Redis 2.6 by default slaves are read-only.
+#
+# Note: read only slaves are not designed to be exposed to untrusted clients
+# on the internet. It's just a protection layer against misuse of the instance.
+# Still a read only slave exports by default all the administrative commands
+# such as CONFIG, DEBUG, and so forth. To a limited extent you can improve
+# security of read only slaves using 'rename-command' to shadow all the
+# administrative / dangerous commands.
+slave-read-only <%=@slavereadonly%>
+<% end %>
+<% if (@version[:major].to_i == 2 && @version[:minor].to_i >= 8 && @version[:tiny].to_i > 18) or @version[:major].to_i > 2 =%>
```

Comment:
This won't match a version string of `2.9.1`.  I'm pretty sure they're done with normal releases in the 2.x series, but possible security releases may bump minor versions in the future.